website/content/en/preview/faq.md

```diff
@@ -110,14 +110,14 @@ Yes, Karpenter supports provisioning metal instance types when a Provisioner's `
 
 ### How does Karpenter dynamically select instance types?
 
-Karpenter batches pending pods and then binpacks them based on CPU, memory, and GPUs required, taking into account node overhead, VPC CNI resources required, and daemon sets that will be packed when bringing up a new node.
-By default Karpenter uses all available instance types, but it can be constrained in the provisioner spec with the [instance-type](https://kubernetes.io/docs/reference/labels-annotations-taints/#nodekubernetesioinstance-type) well-known label in the requirements section.
+Karpenter batches pending pods and then binpacks them based on CPU, memory, and GPUs required, taking into account node overhead, VPC CNI resources required, and daemonsets that will be packed when bringing up a new node.
+By default Karpenter uses C, M, and R >= Gen 3 instance types, but it can be constrained in the provisioner spec with the [instance-type](https://kubernetes.io/docs/reference/labels-annotations-taints/#nodekubernetesioinstance-type) well-known label in the requirements section.
 After the pods are binpacked on the most efficient instance type (i.e. the smallest instance type that can fit the pod batch), Karpenter takes 59 other instance types that are larger than the most efficient packing, and passes all 60 instance type options to an API called Amazon EC2 Fleet.
-The EC2 fleet API attempts to provision the instance type based on a user-defined allocation strategy.
+The EC2 fleet API attempts to provision the instance type based on an allocation strategy.
 If you are using the on-demand capacity type, then Karpenter uses the `lowest-price` allocation strategy.
-So fleet will provision the lowest price instance type it can get from the 60 Karpenter passed it.
+So fleet will provision the lowest priced instance type it can get from the 60 instance types Karpenter passed the API.
```

Comment:
```suggestion
So fleet will provision the lowest priced instance type it can get from the 60 instance types Karpenter passed to the EC2 fleet API.
```